_data/en/settings.yml

```diff
@@ -228,7 +228,7 @@ nav:
   anchor_to_top: Back to top
   become_a_partner: Become a partner
   contact_us:
-    title: Contact us
+    title: Business Inqueries
```

Comment:
Should this be "inquiries", rather than "inqueries" ? (Same applies throughout)

Reply:
Yes - 😭 I'm so bad at spelling. It's been updated <3